### ISSUE TITLE
Cone dystrophy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11405]
ERN-EYE -- need to discuss what to do with this term. Suggestion was "reclassify"

**Comment by pnrobinson** [comment]
Suggestion. Include a parent term 
Pigmentary retinopathy
Child term 
[Cone-rod dystrophy](https://hpo.jax.org/browse/term/HP:0000548)
Probably we do not need an additional term with just "Cone" dystrophy"

Def: Cone rod dystrophies (CRDs) are inherited retinal dystrophies that belong to the group of pigmentary retinopathies. In contrast to typical retinitis pigmentosa (RP), also called the rod cone dystrophies (RCDs) resulting from the primary loss in rod photoreceptors and later followed by the secondary loss in cone photoreceptors, CRDs reflect the opposite sequence of events. CRD is characterized by primary cone involvement, or, sometimes, by concomitant loss of both cones and rods that explains the predominant symptoms of CRDs: decreased visual acuity, color vision defects, photoaversion and decreased sensitivity in the central visual field, later followed by progressive loss in peripheral vision and night blindness. We recommend that specific HPO terms for these anomalies be used, but this term is kept for cases where more specific informationis not available.
PMID:17270046

**Comment by pnrobinson** [comment]
move up to direct child of retinal dystrophy
